src/utils/index.ts

```diff
@@ -8,6 +8,5 @@ export { default as getTextFromRS } from './getTextFromRS';
 export { default as toBuff } from './toBuff';
 export { default as blobCache } from './blobCache';
 export { default as isViewableText } from './isViewableText';
-export { default as intoStream } from './intoStream';
```

Comment:
Unused + is the only thing depending on `web-streams-node`, so we can drop it and the npm lib (which itself requires a bunch of polyfills to support)